pkg/automationconfig/automation_config_builder.go

```diff
@@ -303,6 +317,17 @@ func (b *Builder) Build() (AutomationConfig, error) {
 			AuthSchemaVersion:           5,
 		}
 
+		if b.logRotate != nil {
+			if b.systemLog == nil {
+				return AutomationConfig{}, fmt.Errorf("configurting logRotate without systemlog will not work")
```

Comment:
nit: configuring*

Reply:
Indeed! But I needed to change the whole logic because enterprise code relies on the processModification function pattern, which basically breaks everything. I needed to make some changes due to this 😓